eth_crowdfund_be/eth_crowdfund_api/src/config.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+class Development(object):
+    """
+    Development environment configuration
+    """
+    DEBUG = True
+    TESTING = False
+    JWT_SECRET_KEY = os.getenv('JWT_SECRET_KEY')
+    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:password@localhost/eth_crowdfund_api_db"
```

Comment:
This is not 'secure' at the moment. We would want to use an environment variable in the future